app/Platform/src/Http/Controllers/Auth/Social/LinkedInAuthController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Platform\Http\Controllers\Auth\Social;
+
+use App\Http\Controllers\Controller;
+use App\Platform\Models\User;
+use Knuckles\Scribe\Attributes\Endpoint;
+use Knuckles\Scribe\Attributes\Group;
+use Laravel\Socialite\Facades\Socialite;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+class LinkedInAuthController extends Controller
+{
+    #[Endpoint(title: 'Redirect', description: 'This endpoint redirect to the LinkedIn SignIn Form')]
+    #[Group('Platform')]
+    public function redirect(): RedirectResponse|\Illuminate\Http\RedirectResponse
+    {
+        return Socialite::driver('linkedin')->stateless()->redirect();
```

Comment:
One thing that I forgot to mention in the other endpoints is that we are building an API so not sure if redirecting the visitor from the API is a good idea. Maybe the redirect URL needs to be returned? Investigate this and make the appropriate changes for both endpoints for all social logins.